src/NoFrixion.MoneyMoov/Enums/PaymentProcessorsEnum.cs

```diff
@@ -73,8 +73,13 @@ public enum PaymentProcessorsEnum
     BitcoinTestnet,
 
     /// <summary>
-    /// Agency banking supplier.
+    /// Banking Circle standard accounts.
     /// </summary>
-    BankingCircle
+    BankingCircle,
+
+    /// <summary>
+    /// Banking Circle agency banking accounts.
+    /// </summary>
+    BankingCircleAgency

```

Comment:
Does standard and agency use a different API's that we need different processors?

Reply:
Yep.